docker/importer/importer.py

```diff
@@ -700,8 +700,9 @@ def _process_updates_rest(self, source_repo: osv.SourceRepository):
         # No need to log a full error, as this is expected result.
         logging.info('Entry does not have an OSV entry: %s', vuln.id)
         continue
-      except Exception:
-        logging.error('Failed to parse %s', vuln.id)
+      except Exception as e:
+        logging.error('Failed to parse %s: error type: %s, details: %s',
```

Comment:
https://docs.python.org/3/library/logging.html#logging.Logger.exception might be more appropriate here?

```suggestion
        logging.excecption('Failed to parse %s: error type: %s, details: %s',
```